localization/lk_optical_flow/src/lk_optical_flow.cc

```diff
@@ -25,6 +25,8 @@
 #include <opencv2/video/tracking.hpp>
 #include <ros/ros.h>
 
+#define CV_FONT_NORMAL CV_RAND_NORMAL
```

Comment:
I don't think this is what you want to do... it probably compiles since they are both ints but I think they are unrelated things, one is a font weight and one is a normal distribution?

Reply:
Oh wow, I got tricked by the:
/home/mgouveia/ros_ws/astrobee/localization/lk_optical_flow/src/lk_optical_flow.cc:221:17: error: ‘CV_FONT_NORMAL’ was not declared in this scope; did you mean ‘CV_RAND_NORMAL’?